dateutil/parser/_parser.py

```diff
@@ -895,7 +899,7 @@ def _parse_numeric_token(self, tokens, idx, info, ymd, res, fuzzy):
         elif idx + 2 < len_l and tokens[idx + 1] == ':':
             # HH:MM[:SS[.ss]]
             res.hour = int(value)
-            value = Decimal(tokens[idx + 2])  # TODO: try/except for this?
+            value = self._to_decimal(tokens[idx + 2])  # TODO: try/except for this?
```

Comment:
Shall we remove `TODO` comment from here? Since we are covering this in `_to_decimal`

Reply:
Well, `_to_decimal` can still raise a `ValueError`, and that `TODO` predates the switch to `Decimal`:

https://github.com/dateutil/dateutil/commit/3b43612c35c7d0918ae96d132a698f7e19c7cdff#diff-4b65e7edb2b73639d2d3360fa8d05343L935

That said, I don't really know what it refers to. I'll leave it in for now.